.projenrc.js

```diff
@@ -7,12 +7,12 @@ const releaseWorkflowName = `release-kubectl-v${SPEC_VERSION}`;
 const defaultReleaseBranchName = `kubectl-v${SPEC_VERSION}/main`;
 
 const project = new awscdk.AwsCdkConstructLibrary({
-  author: 'Amazon Web Services',
+  author: 'Amazon Web Services, Inc.',
   cdkVersion: '2.0.0',
   name: `@aws-cdk/asset-kubectl-v${SPEC_VERSION}`,
-  description: 'An Asset construct that contains kubectl, for use in Lambda Layers',
+  description: 'A library that contains the kubectl, as an AssetSource, for use in Lambda Layers',
```

Comment:
AssetSource doesn't actually have a definition. Let's simplify this:
```suggestion
  description: 'A library that contains kubectl for use in Lambda Layers',
```